### DISCUSSION TITLE
[FEATURE] 备份和恢复docker容器

### DISCUSSION THREAD

**Discussion by linktomarkdown** [1Panel-dev/1Panel#6343] [Ideas]
### 1Panel 版本

v1.10.6

### 请描述您的需求或者改进建议

我自己启动还有拉取的docker镜像和容器,能不能一起备份,因为商店的很多我用不到或者我用到了也是改了东西然后构建的,但是备份恢复快照好像自己部署的没有一起备份,希望能增加

### 请描述你建议的实现方案

跟备份商店的应用一样备份,或者让我自己可以将自己的容器转换为私有商店应用就可以了

**Comment by wanghe-fit2cloud** [comment]
容器页面现在支持直接基于容器创建镜像的功能，如图：
<img width="1262" alt="image" src="https://github.com/user-attachments/assets/8703aa91-2ed6-486b-8f7b-444d226e41e7">
